index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Jay Melfah</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/a738fb17fd.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+
+</head>
+<body>
+  <!-- toolbar -->
+  <nav class="nav_bar">
+    <ul class="nav_menu">
+      <li><span class="name">Jay Melfah</span></li>
+      <li><span class="hamburger">☰</span></li>    
+    </ul>
+  </nav>   
+
+  <!-- Headline section -->
+  
+  <header class="container">
+    <div class="intro">
+      <p>Hey There.</p>
+      <p>I'm Jay</p>
+    </div>
```

Comment:
- [ ] As this is the main text of your project, it would be better to make it as a ```<h1>``` tag instead of ```<p>``` tags. Kindly make this a h1 and not paragraph tags and also kindly do not make two h1 tags for this section. Kindly note that h1 tags are only required to be one per page.